pkg/storkctl/action.go

```diff
@@ -7,21 +7,24 @@ import (
 
 	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
 	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/portworx/sched-ops/task"
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/kubectl/pkg/cmd/util"
 )
 
 const (
-	failoverCommand             = "failover"
-	nameTimeSuffixFormat string = "2006-01-02-150405"
+	failoverCommand                    = "failover"
+	nameTimeSuffixFormat string        = "2006-01-02-150405"
+	actionWaitTimeout    time.Duration = 10 * time.Minute
+	actionWaitInterval   time.Duration = 10 * time.Second
 )
 
 func newPerformCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
 	performCommands := &cobra.Command{
 		Use:   "perform",
-		Short: "Perform actions",
+		Short: "perform actions",
```

Comment:
I thought you wanted to change this 'perform' -> 'do'

Reply:
I do want to change. I had reverted it based on what you had said a while ago. I will revert my revert now so that it is `do`